@types/express-fileupload 1.2.0 - All declarations of 'files' must have identical modifiers

Is something wrong with CI or publishing infrastructure?

- Check the [Infrastructure status updates](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/44317) thread for known issues.
- Leave a message on the [DefinitelyTyped Gitter](http://gitter.im/DefinitelyTyped/DefinitelyTyped).

If you know how to fix the issue, make a pull request instead.

- [X] I tried using the `@types/xxxx` package and had problems.
- [X] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [X] I understand that `@types` package versions [do not follow SemVer](https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library).
- [X] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).
- [X] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @Naktibalda @silkimen @vosatom @peterblazejewicz 
  
The library works fine in my project it only occurs when install [express-openapi-validator](https://github.com/cdimascio/express-openapi-validator).

When running `tsc --build`:

```bash
node_modules/@types/express-fileupload/index.d.ts:14:13 - error TS2687: All declarations of 'files' must have identical modifiers.

14             files?: fileUpload.FileArray;
               ~~~~~

node_modules/@types/multer/index.d.ts:57:13 - error TS2687: All declarations of 'files' must have identical modifiers.

57             files: {
               ~~~~~

node_modules/@types/multer/index.d.ts:57:13 - error TS2717: Subsequent property declarations must have the same type.  Property 'files' must be of type 'FileArray | undefined', but here has type '{ [fieldname: string]: File[]; } | File[]'.

57             files: {
               ~~~~~

  node_modules/@types/express-fileupload/index.d.ts:14:13
    14             files?: fileUpload.FileArray;
                   ~~~~~
    'files' was also declared here.


Found 3 errors.
```

--- a/types/multer/index.d.ts
+++ b/types/multer/index.d.ts
@@ -49,12 +49,12 @@ declare global {
 
         interface Request {
             /** `Multer.File` object populated by `single()` middleware. */
-            file: Multer.File;
+            file?: Multer.File;
             /**
              * Array or dictionary of `Multer.File` object populated by `array()`,
              * `fields()`, and `any()` middleware.
              */
-            files: {
+            files?: {
                 [fieldname: string]: Multer.File[];
             } | Multer.File[];
         }

--- a/types/multer/multer-tests.ts
+++ b/types/multer/multer-tests.ts
@@ -18,9 +18,11 @@ assert.strictEqual(upload.constructor.name, 'Multer');
 const app = express();
 
 app.post('/profile', upload.single('avatar'), (req: express.Request, res: express.Response, next: express.NextFunction) => {
+    req.file; // $ExpectTYpe File | undefined
 });
 
 app.post('/photos/upload', upload.array('photos', 12), (req: express.Request, res: express.Response, next: express.NextFunction) => {
+    req.files; // $ExpectType { [fieldname: string]: File[]; } | File[] | undefined
 });
 
 const cpUpload = upload.fields([{ name: 'avatar', maxCount: 1 }, { name: 'gallery', maxCount: 8 }]);

